src/Surfnet/Stepup/Identity/Value/DocumentNumber.php

```diff
@@ -21,47 +21,54 @@
 use JsonSerializable;
 use Surfnet\Stepup\Exception\InvalidArgumentException;
 
+use function preg_match;
+use function strval;
+
 final class DocumentNumber implements JsonSerializable
 {
     /**
-     * @var string
+     * @var string|null
      */
     private $documentNumber;
 
     /**
      * @return self
      */
-    public static function unknown()
+    public static function unknown(): self
     {
-        return new self('—');
+        return new self(null);
```

Comment:
Does this not cause problems when an identitie is forgotten (right to be forgotten) and the entry pops up in the audit log? The value that is set for an unknown document number is displayed then and there in the audit log.

Reply:
This is really beyond my understanding... If you can get the regexp to work with this 'funny' character, we can leave this 'as-is'...